lib/manager/actions/user.js

```diff
@@ -146,12 +113,36 @@ export function resendEmailConfirmation (user: ManagerUserState): Promise<boolea
   }
 }
 
-export function receiveTokenAndProfile (authResult: ?{profile: UserProfile, token: string}) {
-  // authResult can be null in case of first-time user
-  if (!authResult) {
-    return logout()
+/**
+ * Sends a password reset email using the
+ * https://auth0.com/docs/api/authentication#change-password endpoint.
+ * Returns a promise with a resulting value of true if the email was successfully sent.
+ */
+export function sendPasswordReset (user: ManagerUserState) {
+  return async function (dispatch: dispatchFn, getState: getStateFn) {
+    if (!user.profile || !user.profile.email) {
+      throw new Error('User profile missing email!')
+    }
+    const response = await dispatch(
+      secureFetch(
+        `https://${AUTH0_DOMAIN}/dbconnections/change_password`,
+        'POST',
+        {
+          client_id: AUTH0_CLIENT_ID,
+          connection: AUTH0_CONNECTION_NAME,
+          email: user.profile.email
+        },
+        true // return the response object itself.
+      )
+    )
+    alert(response.ok
```

Comment:
If `ok` is missing from the response will this return false or will it just fail?

Reply:
It will treated as if sending the email failed.